build/critical.go

```diff
@@ -9,7 +9,9 @@ import (
 // which case panic will be called instead.
 func Critical(v ...interface{}) {
 	s := fmt.Sprintln(v...)
-	os.Stderr.WriteString(s)
+	if Release != "testing" {
+		os.Stderr.WriteString(s)
+	}
```

Comment:
I think this could be:
```go
if DEBUG {
    panic(s)
} else {
    os.Stderr.WriteString(s)
}
```
since `testing` (essentially) implies `DEBUG`, but not the other way around